lib/media/track/mediatrack.js

```diff
@@ -145,8 +145,10 @@ class MediaTrack extends Track {
   _end() {
     this._log.debug('Ended');
     if (this._dummyEl) {
-      this._detachElement(this._dummyEl);
+      this._dummyEl.remove();
+      this._dummyEl.srcObject = null;
```

Comment:
Just want to double check. Since we're not calling detachElement anymore, is setting srcObject to null enough, and we don't need to call `removeTrack`?
```js
    const mediaStream = this._dummyEl.srcObject;
    if (mediaStream instanceof this._MediaStream) {
      mediaStream.removeTrack(this.processedTrack || this.mediaStreamTrack);
    }
```

Reply:
thats correct, removeTrack applies only for elements attached by application - since they may already have a mediaStream, in that case we add/remove track from it.